observability/micrometer/src/test/resources/org/wildfly/extension/micrometer/micrometer-transform-1_1_0.xml

```diff
@@ -0,0 +1,5 @@
+<!--
+  ~ Copyright The WildFly Authors
+  ~ SPDX-License-Identifier: Apache-2.0
+  -->
+<subsystem xmlns="urn:wildfly:micrometer:1.1"/>
```

Comment:
Where is this used? I don't see any transformer tests :-)

Reply:
@kabir Transformer tests added